c/cert/test/rules/ERR32-C/test.c

```diff
@@ -0,0 +1,42 @@
+#include <errno.h>
+#include <signal.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/wait.h>
+
+typedef void (*pfv)(int);
+
+void handler1(int signum) {
```

Comment:
is this COMPLIANT? (currently shows up as NON_COMPLIANT) - picking on the "the corresponding signal handler obtained a SIG_ERR return from a call to the signal function." wording of the rule. what if errno is accessed on not that type of exit status from the signal function called in the signal handler...?
```
void handler1(int signum) {
  pfv old_handler = signal(signum, SIG_DFL);
  if (old_handler == SIG_ERR) {
    perror(""); // NON_COMPLIANT
  } else {
    perror(""); // COMPLIANT?
  }
}
```